### PR TITLE
Feature build UI

### PR DESCRIPTION
**Pull request by Liimmaa**
1. Import all required asset files.
2. Add 3 screens embedded in this UI feature:
- The home page with a navigation view and tab view embedded.
- Clicking on the 'trending news' carousel at the top of the home page takes you to the detail page.
- Clicking on the search icon on the tab view takes you to the search page.

simulator_screenshot_CC9D539C-97DD-46A0-9DB3-74D2D1B9D151

simulator_screenshot_87FA6C20-25B6-420C-93D8-0379E941BDC9

simulator_screenshot_F6716E5B-D7F3-4B48-A391-B5E8950856C8

### CODE REVIEW

File: iOSLearningApp/DetailView.swift [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        VStack {
+            header
+            newsDetail
+        }
+        .padding()
+        .ignoresSafeArea()
+        .navigationBarHidden(true)
+    }
+    var header: some View {
+        NavigationBar(navigationTitle: "DETAIL", rightIcon1: "square.and.arrow.up", rightIcon2: "bookmark")
```
Comment (Gtwatt1) [inline iOSLearningApp/DetailView.swift:21]: please create a constant file.

Comment (Gtwatt1) [inline iOSLearningApp/DetailView.swift]: please create a constant file for the string to make localisation easier

Comment (Gtwatt1) [reply iOSLearningApp/DetailView.swift]: check out this library https://github.com/mac-cain13/R.swift/ it makes import of assets easier

File: iOSLearningApp/DetailView.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            .padding(.top, 25)
+    }
+    var newsDetail: some View {
```
Comment (Gtwatt1) [inline iOSLearningApp/DetailView.swift]: no spacing, did you use swiftlint?

File: iOSLearningApp/DetailView.swift [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        ScrollView {
+            VStack(alignment: .leading, spacing: 25) {
+                imageReader
+                newsTitle
+                newsOutlet
+                newsSummary
+            }
+        }
+    }
+    var imageReader: some View {
```
Comment (Gtwatt1) [inline iOSLearningApp/DetailView.swift]: private? all variables could be private right?

File: iOSLearningApp/NavigationBar.swift [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+//
+//  Created by Chioma Amanda Mmegwa  on 11/10/2022.
+//
+
+import SwiftUI
+
+struct NavigationBar: View {
+    var navigationTitle: String
+    var rightIcon1: String?
+    var rightIcon2: String?
```
Comment (Gtwatt1) [inline iOSLearningApp/NavigationBar.swift]: could be PRivates?

**Comment by Gtwatt1** [comment]
looking great .

File: iOSLearningApp/DetailView.swift (hunk 2 above)
Comment (Liimmaa) [reply iOSLearningApp/DetailView.swift]: Yes, I did. Does not throw up any errors. If i put any spaces, swiftlint flags it.

File: iOSLearningApp/NavigationBar.swift (hunk 4 above)
Comment (Liimmaa) [reply iOSLearningApp/NavigationBar.swift]: If I made them privates, I wont be able to use my custom NavigationBar objects outside this file.

**Pull request merged**